plugins/inputs/http_response/http_response.go

```diff
@@ -16,13 +16,15 @@ import (
 	"time"
 	"unicode/utf8"
 
-	"github.com/benbjohnson/clock"
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/internal"
 	"github.com/influxdata/telegraf/plugins/common/cookie"
 	"github.com/influxdata/telegraf/plugins/common/tls"
 	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/benbjohnson/clock"
+	"github.com/seancfoley/ipaddress-go/ipaddr"
```

Comment:
This should go before the telegraf imports...

Reply:
Right, my bad.